app/controllers/collections_controller.rb

```diff
@@ -8,6 +8,10 @@ def index
 
   # GET /collections/the-uid.json
   def show
+    # respond_to do |format|
```

Comment:
This controller action is still not rendering the json.jbuilder template, instead the raw json is being rendered instead. I'll remove the commented out code but I wanted to share my approach with you, to see if you had any advice.